Misc/NEWS.d/next/Library/2020-04-24-01-27-08.bpo-38061.cdlkMz.rst

```diff
@@ -0,0 +1,6 @@
+On FreeBSD, ``os.closerange(fd_low, fd_high)`` now calls ``closefrom(fd_low)``
+if *fd_high* is greater than or equal to ``sysconf(_SC_OPEN_MAX)``.
+
+Initial patch by Ed Maste (emaste), Conrad Meyer (cem), Kyle Evans (kevans)
+and Kubilay Kocak (koobs):
```

Comment:
Can remove this, full credit to Ed, Conrad & Kyle

Reply:
Even if you didn't write the patch itself, you helped to get it in FreeBSD package, and so you deserve to be credited for this work ;-) Thanks @koobs!